index.html

```diff
@@ -0,0 +1,121 @@
+<!DOCTYPE html>
+<html lang="en">
+  <head>
+    <meta charset="UTF-8" />
+    <meta http-equiv="X-UA-Compatible" content="IE=edge" />
+    <meta name="viewport" content="width=device-width, initial-scale=1.0" />
+    <title>My Portfolio</title>
+    <link rel="stylesheet" href="./style.css" />
+    <link
+      rel="stylesheet"
+      href="https://cdnjs.cloudflare.com/ajax/libs/font-awesome/6.1.0/css/all.min.css"
+      integrity="sha512-10/jx2EXwxxWqCLX/hHth/vu2KY3jCF70dCQB8TSgNjbCVAC/8vai53GfMDrO2Emgwccf2pJqxct9ehpzG+MTw=="
+      crossorigin="anonymous"
+      referrerpolicy="no-referrer"
+    />
+    <link rel="icon" href="./assets/coding.png" type="image/x-icon" />
+  </head>
+  <body>
+    <header>
+      <div class="nav-desktop"></div>
+      <div class="nav-mobile">
+        <h2>Welcome</h2>
+        <div class="btn"><i class="fa-solid fa-bars"></i></div>
+      </div>
+    </header>
+
+    <section class="welcome-section">
+      <div class="welcome-header-text">
+        <h2>Hey there. I'm Kano</h2>
+        <h2 class="second-header">I am software developer</h2>
+      </div>
+      <div class="intro-text">
+        <p>
+          I can help you build a product , feature or website Look through some
+          of my work and experience! If you like what you see and have a project
+          you need coded, don’t hestiate to contact me.
+        </p>
+      </div>
+
+      <ul class="social-links">
+        <img
+          src="./assets/social-media/github.svg"
+          alt="github social media icon "
+        />
+        <img
+          src="./assets/social-media/Linkedin.svg"
+          alt="github social media icon "
+        /><img
+          src="./assets/social-media/twitter.svg"
+          alt="twitter social media icon "
+        /><img
+          src="./assets/social-media/angellist.svg"
+          alt="angellist social media icon "
+        />
+        <img
+          src="./assets/social-media/medium.svg"
+          alt="Medium social media icon "
+        />
```

Comment:
- [ ] Good job placing these social media icons in a `ul` tag but it would be best to place each of the images in an `li` tag. Kindly rectify this issue 😄